example/src/dog-workspace.ts

```diff
@@ -55,6 +55,15 @@ export const dogWorkspace = {
         },
       },
     },
+    {
+      name: "Redirecting HTML response",
```

Comment:
Title case for route? 🤷 
```suggestion
      name: "Redirecting HTML Response",
```